src/pages/public/signup/index.js

```diff
@@ -21,33 +15,14 @@ const SignUp = () => {
   };
 
 
-  const submitForm = event => {
-    // console.log(
-    //   arrayOfEmail.find(item => {
-    //     console.log(email + "the email");
-    //     item == email;
-    //   })
-    // );
-    const localData = JSON.parse(localStorage.getItem("user"));
-    let flag = false;
-    localData.map(item => {
-      return (item.Email === email ? flag = true : console.log("flag"))
-    });
+  const submitForm = (event) => {
```

Comment:
When using Form, Try using formik with only required or proper validation for possible fields.